packages/llvm/project.bri

```diff
@@ -0,0 +1,110 @@
+import * as std from "std";
+import { cmakeBuild } from "cmake";
+import { gitCheckout } from "git";
+import python from "python";
+import nushell from "nushell";
+
+export const project = {
+  name: "llvm",
+  version: "20.1.1",
+};
+
+const source = gitCheckout(
+  Brioche.gitRef({
+    repository: "https://github.com/llvm/llvm-project.git",
+    ref: `llvmorg-${project.version}`,
+  }),
+);
+
+export default function llvm(): std.Recipe<std.Directory> {
+  let llvm = cmakeBuild({
+    source,
+    path: "llvm",
+    set: {
+      LLVM_ENABLE_PROJECTS: "clang;clang-tools-extra;lld;lldb",
+      CMAKE_BUILD_TYPE: "Release",
+    },
+    env: {
+      CMAKE_BUILD_PARALLEL_LEVEL: "16",
+    },
+    dependencies: [std.toolchain(), python()],
+  });
+
+  llvm = std.setEnv(llvm, {
+    LIBRARY_PATH: { append: [{ path: "lib" }] },
+    CPATH: { append: [{ path: "include" }] },
+  });
+
+  // Remove a bunch of `*.tmp*` files left over in the build
+  // TODO: Figure out where these temp files are coming from!
+  llvm = std.runBash`
+    cd "$BRIOCHE_OUTPUT"
+    find . -name '*.tmp*' -delete
+  `
```

Comment:
Still no clue why these leftover temp files are here... maybe it's a result of our custom CMake patches? This should definitely get fixed